CONTRIBUTORS.txt

```diff
@@ -78,6 +85,7 @@ needs to be listed here.
 - Tim Condon <0xtimc@gmail.com>
 - Tim Kientzle <tkientzle@apple.com>
 - Tomer Doron <tomerd@apple.com>
+- Trevör Anne Denise <adtrevor@users.noreply.github.com>
```

Comment:
@adtrevor if you would like me to add a proper email address here, please reply with the email address you'd like to have here.